scripts/obtain/dotnet-install.sh

```diff
@@ -161,9 +161,12 @@ get_current_os_name() {
             echo "linux"
             return 0
         fi
+    elif [[ $uname == MINGW* ]]; then
+        echo "win"
+        return 0
     fi
 
-    say_err "OS name could not be detected: $ID.$VERSION_ID"
```

Comment:
This didn't actually work as $ID and $VERSION_ID are undefined in the error scenario